hidtest/test.c

```diff
@@ -134,24 +134,27 @@ int device_callback(
 {
 	(void)user_data;
 
-    if (event & HID_API_HOTPLUG_EVENT_DEVICE_ARRIVED)
-        printf("Handle %d: New device is connected: %s.\n", callback_handle, device->path);
-    else
-        printf("Handle %d: Device was disconnected: %s.\n", callback_handle, device->path);
-
-    printf("type: %04hx %04hx\n  serial_number: %ls", device->vendor_id, device->product_id, device->serial_number);
-    printf("\n");
-    printf("  Manufacturer: %ls\n", device->manufacturer_string);
-    printf("  Product:      %ls\n", device->product_string);
-    printf("  Release:      %hx\n", device->release_number);
-    printf("  Interface:    %d\n", device->interface_number);
-    printf("  Usage (page): 0x%hx (0x%hx)\n", device->usage, device->usage_page);
-    printf("\n");
-
-    //if (device->product_id == 0x0ce6)
-    //    return 1;
-
-    return 0;
+	if (event & HID_API_HOTPLUG_EVENT_DEVICE_ARRIVED)
```

Comment:
thanks
maybe a separate PR with the fix into master?

Reply:
No need - it was done the same way in the Windows Mutex PR (was asked to fix indentation and also had troubles with no `fflush()`), so this change no longer even pops up in the diff

Reply:
oh, I only meant replacing spaces with tabs inside of this function

everything else, obviously, is feature/branch-specific